test/abstract_syntax_tree_location_data.coffee

```diff
@@ -2695,17 +2695,16 @@ test "AST as expected for Try node", ->
             line: 1
             column: 14
       ]
-      # TODO: the location data for the block is wrong (wrong location data on generated INDENT)
-      # start: 4
-      # end: 14
-      # range: [4, 14]
-      # loc:
-      #   start:
-      #     line: 1
-      #     column: 4
-      #   end:
-      #     line: 1
-      #     column: 14
+      start: 3
```

Comment:
This is one character before what might be expected (3 vs 4). This is because when assigning the artificial location for the generated `INDENT` token, it pretends that it's a zero-width token one character after the preceding token

So in this example (`try cappuccino`), rather than assigning the block the exact location data of `cappuccino`, it's assigning the block the location data including the preceding space (` cappuccino`). I think this is fine, if we end up seeing any issues with this we can look more closely at other strategies for assigning the `INDENT` location data I guess